### DISCUSSION TITLE
please help

### DISCUSSION THREAD

**Discussion by Al5x404-cat** [HaxeFlixel/flixel#3039] [Q&A]
so either im stupid or it litteraly wont let me. on the haxeflixer 2d doc, in hello world it tells me to use cd C:\Users\<User>\Projects
i cant use it at all. help?

**Comment by Geokureli** [comment]
https://en.wikipedia.org/wiki/Cd_(command)

`cd` is a terminal command that changes the directory of the terminal, the doc is likely telling you to use itso that you can run terminal commands in the project, since flixel using lime commands to build and run games

**Reply by Al5x404-cat** [reply]
i am trying but it wont work though
just says access denied

**Reply by Geokureli** [reply]
can you paste exactly what you're trying and the result you're getting

**Reply by Al5x404-cat** [reply]
"C:\Users\Blake>cd C:\Users\Blake\Projects
The system cannot find the path specified"
if it cant find it, how do i create it?

**Reply by Geokureli** [reply]
I assume you're using this: https://haxeflixel.com/documentation/hello-world/
You don't need to create the template in the projects folder, thats just a common place where people put their projects. I reccommend puting a new folder wherever you want your haxe projects to go, simply open up the file explorer create a projects folder or haxe folder and then simply type `cd ` in your terminal and then drag the folder into the terminal and it will insert the full path, press enter to run the cd command and then you can run `flixel tpl -n "HelloWorld"`
